database/migrations/2018_06_22_160305_add_source_details_column_to_signup_table.php

```diff
@@ -14,7 +14,7 @@ class AddSourceDetailsColumnToSignupTable extends Migration
     public function up()
     {
         Schema::table('signups', function (Blueprint $table) {
-            $table->text('source_details')->after('source')->comment('Extra details about the signup source, like referral source from url query string');
+            $table->string('source_details')->after('source')->nullable()->comment('Extra details about the signup source, like referral source from url query string');
```

Comment:
This needs to be a new migration if you want to make a change to this column! Once a migration has run, it doesn't get run again (unless you run `php artisan migrate:rollback` to undo it, like when testing on your local).

Reply:
@DFurnes ah ok! So make this a new migration instead of just editing here? 

Not sure if you saw the above but here's what I was thinking:
>  this migration is just on thor and my local so I thought instead of creating a new migration, it'd be better to just update this one. On my local, I deleted the migration in the migrations table and the source_details column and re-ran the migration. Is it ok to do this for thor too the same way?

Reply:
ok sounds good to me, thanks @DFurnes !